src/gui/menu.cpp

```diff
@@ -44,15 +44,18 @@
 #include "gui/menu_manager.hpp"
 #include "gui/mousecursor.hpp"
 #include "math/util.hpp"
+#include "supertux/error_handler.hpp"
 #include "supertux/gameconfig.hpp"
 #include "supertux/globals.hpp"
 #include "supertux/resources.hpp"
 #include "video/drawing_context.hpp"
 #include "video/renderer.hpp"
+#include "video/surface.hpp"
 #include "video/video_system.hpp"
 #include "video/viewport.hpp"
 
-#include "supertux/error_handler.hpp"
+const Sizef Menu::s_preview_size(426.f, 240.f);
```

Comment:
Hardcoding the preview size squishes pictures that are too big. Any reason as to why you're doing this?

Reply:
So the image and the menu could both fit on the screen. Maybe I should check the resolution before determining this though.